node/taskfiles/taskfile.local.yml

```diff
@@ -9,3 +9,7 @@ tasks:
     dotenv: [".env"]
     cmds:
       - go test ./pkg/admin/tests -v
+  test:
```

Comment:
You could use dependencies instead. It is more clear.

```
test:
  deps:
    - test-db
    - test-admin
```

Reply:
deps are to define tasks that depend on each other, but in this case, it's not depending on each other, but I wanted to declare a task that runs all test tasks.
https://taskfile.dev/usage/#task-dependencies

in official document, it is recommending to use `cmds` to run other tasks
https://taskfile.dev/usage/#calling-another-task